package.json

```diff
@@ -18,8 +18,8 @@
     "deep-freeze": "0.0.1",
     "js-yaml": "3.11.0",
     "lodash": "4.17.10",
-    "patternfly": "3.45.3",
```

Comment:
I think the PF update should go into a separate PR.

Reply:
We use to do that in the backend I don't know why we don't do in the front, for me is fine, I should  change in https://github.com/kiali/kiali-ui/pull/407 too.